packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -396,7 +407,7 @@ class TxForm extends Component {
 
       // If the gas hasn't been calculated yet, then we don't show any errors,
       // just wait a bit more
-      if (!this.estimatedTxFee(values)) {
+      if (!this.isEstimatedTxFee(values)) {
```

Comment:
replace with



```js

// If the gas hasn't been calculated yet, then we don't show any errors,
// just wait a bit more
if (!this.isEstimatedTxFee(values)) {
  return "Estimating gas...";
}
```

if the gas wasn't estimated yet, then the form isn't valid

Reply:
the only con is that this will show up as a disabled "Send" button rather than a "Verifying" button (while the gas is being estimated), but since this is for a short amount of time I think it's ok

Reply:
Yeah I think that's fine as a workaround for now, we can see later if that's a problem on mainnet later.